tutor/src/screens/assignments/details.js

```diff
@@ -32,10 +34,73 @@ const StyledTextInput = styled(TextInput)`
   max-width: 48rem;
 `;
 
-const StyledSelect = styled(Select)`
-  min-width: 24rem;
+const StyledDropdown = styled(Dropdown)`
+  display: inline-flex;
 `;
 
+const StyledToggle = styled(Dropdown.Toggle)`
+  &&& {
+    border: 1px solid ${colors.forms.borders.light};
```

Comment:
I'm not familiar with the `&&&` syntax.  Is that just to make the rule specific enough?

you might also be able to nest under body or something else like:
```
  body & {
    border: 1px solid red;
  }
```

Reply:
Yeah it's a specificity thing, components from Bootstrap seem to be especially finicky for overriding.